src/ShortidType.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
I don't see the need for this.

Reply:
I don't think there is a *need* for strict_types at all. But from my point of view in times of PHP 7 and even more with PHP 8 one should have a good reason to not use strict_types by default everywhere. (Not enabling strict_types feels like using the quirks mode in IE6.) I don't want to open a debate on principles about this here, so I'll remove it with the next commit.